doc/alarm.md

```diff
@@ -204,33 +221,34 @@ To use an custom implementation of the webhook sender, you can define new bean i
     </bean>
 ```
 
-To enable the webhook sender service, you need to set **webhook.enable** and **webhook.receiver.url** in [batch-root.properties](https://github.com/pinpoint-apm/pinpoint/blob/master/batch/src/main/resources/batch-root.properties).  webhook.enable means turning on or off the webhook alarm service and webhook.receiver.url means to which a receiver receive the alarm message.
+### 2.1.2) Configuring MYSQL
 
-```properties
-# webhook config
-webhook.enable=true
-webhook.receiver.url=http://www.webhookexample.com/alarm/
-```
+**step 1**
 
-Lastly, you should set the type of the alarm rule to webhook in the Pinpoint alarm page.
+Prepare MYSQL Instance to persist the alarm service metadata.
 
-### 2.1.2) Configuring MYSQL
+**step 2**
 
-Set up a MYSQL server and configure connection information in *jdbc.properties* file.
+Set up a MYSQL server and configure connection information in [jdbc-root.properties](https://github.com/pinpoint-apm/pinpoint/blob/master/batch/src/main/resources/jdbc-root.properties) file.
 
-```
+```properties
 jdbc.driverClassName=com.mysql.jdbc.Driver
 jdbc.url=jdbc:mysql://localhost:13306/pinpoint?characterEncoding=UTF-8
 jdbc.username=admin
 jdbc.password=admin
 ```
-Create tables by running - *[CreateTableStatement-mysql.sql](https://github.com/pinpoint-apm/pinpoint/blob/master/web/src/main/resources/sql/CreateTableStatement-mysql.sql)*, *[SpringBatchJobReositorySchema-mysql.sql](https://github.com/pinpoint-apm/pinpoint/blob/master/web/src/main/resources/sql/SpringBatchJobRepositorySchema-mysql.sql)*
+**step 3**
+
+Create tables for the alarm service. Use below DDL files.
+
+- [CreateTableStatement-mysql.sql](https://github.com/pinpoint-apm/pinpoint/blob/master/web/src/main/resources/sql/CreateTableStatement-mysql.sql) 
+- [SpringBatchJobReositorySchema-mysql.sql](https://github.com/pinpoint-apm/pinpoint/blob/master/web/src/main/resources/sql/SpringBatchJobRepositorySchema-mysql.sql)
 
 
 ### 2.1.3) How to execute pinpoint-batch
 
 The pinpoint-batch project is based on spring boot and can be executed with the following command.
-After build, the executable file is placed under the target/deploy folder of the pinpoint-batch module.
```

Comment:
is this necessary change?

Reply:
I think using the word 'module' is not consistent with the other lines.
The word isn't used except for this line.

Reply:
okay, if you say so.